TestFoundation/TestJSONEncoder.swift

```diff
@@ -401,6 +401,68 @@ class TestJSONEncoder : XCTestCase {
         test_codingOf(value: URL(string: "https://swift.org")!, toAndFrom: "\"https://swift.org\"")
     }
 
+
+    // UInt and Int
+    func test_uintMinMax() {
```

Comment:
Can the naming of this be consistent with the other tests, so `test_codingOfUIntMinMax()`?

Also we should add this testcase on the Darwin side too: https://github.com/apple/swift/blob/master/test/stdlib/CodableTests.swift